ui/page/fileListPublished/view.jsx

```diff
@@ -4,45 +4,86 @@ import Button from 'component/button';
 import ClaimList from 'component/claimList';
 import Page from 'component/page';
 import Paginate from 'component/common/paginate';
-import { PAGE_SIZE } from 'constants/claim';
+import { PAGE_PARAM, PAGE_SIZE_PARAM } from 'constants/claim';
 import WebUploadList from 'component/webUploadList';
 import Spinner from 'component/spinner';
+import Card from 'component/common/card';
 
 type Props = {
+  uploadCount: number,
   checkPendingPublishes: () => void,
   clearPublish: () => void,
-  fetchClaimListMine: () => void,
+  fetchClaimListMine: (number, number) => void,
   fetching: boolean,
   urls: Array<string>,
-  urlTotal: ?number,
-  history: { replace: string => void },
+  urlTotal: number,
+  history: { replace: string => void, push: string => void },
   page: number,
+  pageSize: number,
 };
 
 function FileListPublished(props: Props) {
-  const { checkPendingPublishes, clearPublish, fetchClaimListMine, fetching, urls, urlTotal } = props;
+  const {
+    uploadCount,
+    checkPendingPublishes,
+    clearPublish,
+    fetchClaimListMine,
+    fetching,
+    urls,
+    urlTotal,
+    page,
+    pageSize,
+  } = props;
+
+  const params = {};
+
+  params[PAGE_PARAM] = Number(page);
+  params[PAGE_SIZE_PARAM] = Number(pageSize);
+
+  const paramsString = JSON.stringify(params);
+
   useEffect(() => {
     checkPendingPublishes();
-    fetchClaimListMine();
-  }, [checkPendingPublishes, fetchClaimListMine]);
+  }, [checkPendingPublishes]);
+
+  useEffect(() => {
+    if (paramsString && fetchClaimListMine) {
+      const params = JSON.parse(paramsString);
+      fetchClaimListMine(params.page, params.page_size);
+    }
+  }, [uploadCount, paramsString, fetchClaimListMine]);
```

Comment:
uploadCount is for lbry.tv. 
I could use two different @if TARGETS for two different useEffect hooks, or just leave it - it seems to work.

Reply:
I think it's fine.